github/event_types.go

```diff
@@ -491,8 +491,9 @@ type PullRequestEvent struct {
 
 	// The following fields are only populated by Webhook events.
 	Changes *EditChange `json:"changes,omitempty"`
-	// RequestedReviewer is populated in "review_requested", "review_request_removed" event deliveries. A request for
-	// multiple users to review at once is split into multiple requests, each with a single, different RequestedReviewer.
+	// A request affecting multiple reviewers at once is split into multiple
+	// "review_requested"/"review_request_removed" event deliveries, each with
+	// a single, different RequestedReviewer
```

Comment:
Please add a period to the end of the sentence to make the [resulting Godocs](https://godoc.org/github.com/google/go-github/github) read nicely.

Reply:
This commit removed the first sentence:

```Go
// RequestedReviewer is populated in "review_requested", "review_request_removed" event deliveries.
```

I think we should restore it. Arguably, it's more important, because it tells the reader when `RequestedReviewer` values are populated. The second sentence that elaborates on how requests with multiple reviewers get split is a secondary detail.

Here's my suggested wording:

```Go
// RequestedReviewer is populated in "review_requested", "review_request_removed" event deliveries.
// A request affecting multiple reviewers at once is split into multiple
// such event deliveries, each with a single, different RequestedReviewer.
```

Reply:
@shurcooL @gmlewis Changed, sorry for the rush. In my haste to rapidly push "trivial" commits, I keep making errors (deleting random things w/ cut and paste) and making the pr take longer than if I had just taken the time to read it through. Took a good look through the past few commits, hope it looks all good now.